config/config.toml.example

```diff
@@ -66,6 +66,11 @@ delay-clean-table-lock = 0
 # Maximum number of the splitting region, which is used by the split region statement.
 split-region-max-num = 1000
 
+# enable-pk-is-handle is used to control alter primary key feature. Default is true, indicate the alter primary key feature is disabled.
+# If it is false, we can add the primary key by "alter table", but we may not be able to drop the primary key.
+# In order to support "drop primary key" operation , this flag must be false and the table does not have the pkIsHandle Flag.
```

Comment:
`pkIsHandle` is an internal flag and users cannot know it?

Reply:
Yes, If using the wrong value to `alter primary key`, we will return an error with `enable-pk-is-handle` and `pkIsHandle` information.